device/transport/mqtt/devdoc/mqtt_requirements.md

```diff
@@ -332,12 +292,12 @@ The `getTwinReceiver` method creates a `MqttTwinReceiver` object for the twin re
 
 **SRS_NODE_DEVICE_MQTT_16_046: [** `disableMethods` shall call its callback with an `Error` if an error is received while unsubscribing. **]**
 
-### disableTwin
+### disableTwinDesiredPropertiesUpdates
 
-**SRS_NODE_DEVICE_MQTT_16_062: [** `disableTwin` shall call its callback immediately if the MQTT connection is already disconnected. **]**
+**SRS_NODE_DEVICE_MQTT_16_062: [** `disableTwinDesiredPropertiesUpdates` shall call its callback immediately if the MQTT connection is already disconnected. **]**
 
-**SRS_NODE_DEVICE_MQTT_16_063: [** `disableTwin` shall unsubscribe from the topics for twin messages. **]**
+**SRS_NODE_DEVICE_MQTT_16_063: [** `disableTwinDesiredPropertiesUpdates` shall unsubscribe from the topics for twin messages. **]**
```

Comment:
>shall unsubscribe  [](start = 73, length = 18)

same comment.  Does this function call the mqtt twin object, or does it call the base transport?  I hope it's calling the twin object.

Reply:
twin object. i'll change the requirement.

---
In reply to: [170615308](https://github.com/Azure/azure-iot-sdk-node/pull/236#discussion_r170615308) [](ancestors = 170615308)